packages/playground/src/components/pull-request/pull-request-list.tsx

```diff
@@ -85,33 +86,53 @@ const Title = ({
   success,
   title,
   labels,
-  state
+  state,
+  isDraft,
+  comments
 }: {
+  isDraft?: boolean
   state?: string
   success: boolean
   title: string
+  comments?: number
   labels: { text: string; color: string }[]
 }) => {
   return (
     <div className="flex gap-2 items-center">
-      <Icon
-        size={16}
-        className={cx({ 'text-success': state === 'open' })}
-        name={state === 'open' ? 'pr-open' : success ? 'pr-merge' : state === 'closed' ? 'pr-closed' : 'pr-draft'}
-      />
+      <div className="flex gap-2 flex-wrap justify-start">
+        <Icon
+          size={16}
+          className={cx({
+            'text-success': state === 'open' && !isDraft,
+            'text-tertiary-background': state === 'open' && isDraft,
+            'text-ai': success
+          })}
+          name={
+            state === 'open' && isDraft
```

Comment:
let's create a util for getting icon from pr state. might be re-used in other places

Reply:
will create a util